patches/minecraft/net/minecraft/tags/NetworkTagCollection.java.patch

```diff
@@ -0,0 +1,47 @@
+--- a/net/minecraft/tags/NetworkTagCollection.java
++++ b/net/minecraft/tags/NetworkTagCollection.java
+@@ -9,12 +9,26 @@
+ 
+ public class NetworkTagCollection<T> extends TagCollection<T> {
+    protected final RegistryNamespaced<ResourceLocation, T> field_200044_a;
++   protected final java.util.function.Function<T,Integer> itemToId;
++   protected final java.util.function.Function<Integer,T> idToItem;
+ 
++   public NetworkTagCollection(java.util.function.Predicate<ResourceLocation> isValueKnownPredicateIn, java.util.function.Function<ResourceLocation, T> resourceLocationToItemIn, java.util.function.Function<T, Integer> itemToId, java.util.function.Function<Integer, T> idToItem, String resourceLocationPrefixIn, boolean preserveOrderIn, String itemTypeNameIn) {
```

Comment:
your parameter names are horrible, what is with the 'in' suffix? and 'getX' is a good format for getters...

Reply:
I just took over the mcp names I found elsewhere...
Will change it

Reply:
MCP names are horrible crowd sourced things, pick real names that are appropriate.